region/norfair/east.json

```diff
@@ -423,6 +423,7 @@
                   "requires": [
                     "h_canNavigateHeatRooms",
                     "Plasma",
+                    "canHeroShot",
```

Comment:
I see this now as  `OR:[ Plasma, canHeroShot]`
Plasma's value is in getting around heroShotting.
Hero shot shinespark loses value as a notable with this tech too.